diagrams/README.md

```diff
@@ -0,0 +1,62 @@
+# Design
+
+
+## Core Assumptions
+
+
+1.  Supply is fixed at 1B
+2.  Rewards will never end
```

Comment:
If supply is fixed, can we guarantee rewards never end?

Reply:
right now `METRIC_PER_BLOCK` tracks the emission rate, and it's hardcoded to 4 - but when we get into the Rewards design we will want to put this behind a function which will dynamically calculate the emission rate - OR - we will want to invalidate these assumptions.  It's much more common to have a projected supply over the years and emit based on the network's block-rate to match that projection as close as possible.

Either way, once we get into rewards, we can revisit these!

Reply:
Will rewards be figured out before these contracts are deployed?

Reply:
yes - we need to get 80% of the implementation ready to even be able to book an auditor - and then they'll take at least a month before they can even get started --

Launch will include the token, vesting, staking, the faucet, and question creation